http.js

```diff
@@ -37,6 +37,8 @@ module.exports = function probeHttp(options, _callback) {
           if (length && length.match(/^\d+$/)) {
             result.length = +length;
           }
+
+          result.responseUrl = res.request.uri.href;
```

Comment:
Good point, from looking at the doc it is: https://github.com/request/request/blob/master/request.js#L846, https://github.com/request/request/blob/master/request.js#L194